compiler/ballerina-parser/src/test/resources/expressions/query-expr/query_expr_source_94.bal

```diff
@@ -0,0 +1,5 @@
+function testGroupBy() {
+    var res = from var {price1, price2, name} in orders
+        group by int price = price1 +
+        select name;
+}
```

Comment:
Shall we add a few more tests like,

1. group by var price
2. group by var price1, price2
3. group by price1, var price2
4. group by price1 price2